pkgs/development/tools/build-managers/gup/build.nix.gup

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu
+if [ -n "${GUP_TARGET:-}" ]; then
+	gup --always
+fi
+curl -LSs -o "$1" https://raw.githubusercontent.com/timbertson/gup/master/nix/gup-python.nix
```

Comment:
OK, if you commit to always prevent «`master` contains `build.nix` for the future release, not for the current» as the maintainer on both sides, then I guess that's acceptable.